File: Week3/hojun/migrations/Post.js
const db = require('../models/index');
const sequelize = db.sequelize;
const { DataTypes } = db.Sequelize;

module.exports = async () => {
    const Post = await sequelize.define('Posts', {
        postId: {
            type: DataTypes.INTEGER,
            primaryKey: true,
            autoIncrement: true
        },
        title: DataTypes.STRING,
        content: DataTypes.TEXT,
        date: DataTypes.DATE,
        userId: DataTypes.INTEGER,
        views: DataTypes.INTEGER,
        like_count: DataTypes.INTEGER
    })

    return Post;
}

File: Week3/minju/Domain/Post.java
    package week3.seed.Domain;

    import lombok.Data;

    import javax.persistence.*;
    import java.util.Date;

    @Data
    @Entity
    public class Post {
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        private Integer postId;
        private String title;
        @Column(columnDefinition = "TEXT")
        private String content;
        private Integer views;
        private Integer likeCount;

        @ManyToOne
        @JoinColumn(name = "user_id")
        private User user;

        private Date date;
    }